sources/packages/web/src/assets/css/base.scss

```diff
@@ -57,3 +57,7 @@ html * {
 .p-md-4 {
   padding: 0.5rem !important;
 }
+
+.padding-bottom-30 {
+  padding-bottom: 30px;
```

Comment:
i guess, you can use bootstrap classes instead, https://getbootstrap.com/docs/4.0/utilities/spacing/

Reply:
You are right. I wanted to use the bootstrap classes but used the quick one to check how it looks and it got left.

Reply:
Feedback comment addressed.